deployer/grafana/grafana_tokens.py

```diff
@@ -0,0 +1,210 @@
+import json
+import subprocess
+from base64 import b64encode
+from pathlib import Path
+
+import requests
+import typer
+
+from ..cli_app import app
+from ..utils import print_colour
+from .grafana_utils import get_grafana_url, update_central_grafana_token
+
+REPO_ROOT = Path(__file__).parent.parent.parent
+
+
+def build_service_account_request_headers():
+    """
+    Build the headers needed to send requests to the Grafana service account endpoint.
+
+    Returns:
+        dict: "Accept", "Content-Type", "Authorization" headers
+    """
+    # Dencrypt grafana credentials file
+    grafana_username = "admin"
+    grafana_password_string = subprocess.check_output(
+        [
+            "sops",
+            "--decrypt",
+            REPO_ROOT / "helm-charts/support/enc-support.secret.values.yaml",
+        ]
+    ).decode("utf-8")
+
+    # Parse the sops output to find the admin's username password
+    keyword = "adminPassword: "
```

Comment:
You can use https://github.com/2i2c-org/infrastructure/blob/348a88b436edcb14d104577a8479d9edd9a5ca84/deployer/file_acquisition.py#L72 to read the encrypted file, and it'll handle decryption correctly. We should also definitely use yaml to read the decrypted content, rather than using string parsing I think.

Reply:
Haha, yes! I have used this before. I have no idea why this time my impulse was to reinvent the wheel do everything the ugly manual way 😅 Thanks for raising this